cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -853,7 +850,7 @@ private module Stage2 {
     ap = getApNil(node)
     or
     exists(Node mid, Ap ap0, LocalCc localCc |
-      fwdFlow(mid, cc, argAp, ap0, config) and
+      fwdFlow(mid, pragma[only_bind_into](cc), argAp, ap0, config) and
```

Comment:
Most `getLocalCc`s are just `any()`, so I reckon this is only relevant for stage 4? If so, perhaps move the pragma into `Stage4::getLocalCc`?

Reply:
Sure, I can do that.